src/app/TideIndexApplication.cpp

```diff
@@ -126,10 +126,11 @@ int TideIndexApplication::main(
       "tide-index.peptides.txt").c_str(), NULL, overwrite);
   }
   
-  ofstream* out_decoy_fasta = GeneratePeptides::canGenerateDecoyProteins() ?
+/*  TODO: Reocer the option to generate decoy protien fasta file.

```

Comment:
I'm confused about this TODO. I think the typo is confusing me. What do you want to do with the option?

Also - typo in "protien"

Reply:
I guess, I meant "recover".  The new tide-index does not support an option to produce decoy fasta.